yandex_music/exceptions.py

```diff
@@ -12,6 +12,10 @@ class InvalidBitrateError(YandexMusicError):
     """Класс исключения, вызываемого при попытке загрузки трека с недоступным битрейтом."""
 
 
+class IDMissingError(YandexMusicError):
```

Comment:
we have classes like `TrackId`, `PlaylistId` etc`. pls let's keep it in the same style 

```suggestion
class IdMissingError(YandexMusicError):
```